astrobee/config/robots/bsharp.config

```diff
@@ -24,8 +24,15 @@ robot_imu_drdy_pin = 4
 
 robot_geometry = {
 -- Placeholder transforms, not accurate!
-  hazcam_to_navcam_transform = transform(vec3(0.079356943, 0.00019413532, -0.0019157775), quat4(0.076822296, -0.009733734, 0.99698419, 0.0051110388)),
-  scicam_to_hazcam_transform = transform(vec3(0.0070938664, 0.061024816, -0.11184277), quat4(0.065639737, -0.031237584, 0.99720414, -0.017307023)),
+  nav_cam_to_haz_cam_transform = transform(vec3(0.071421481, -0.00030319673, 0.0018058249), quat4(-0.0030431141, 0.0092646368, 0.99993195, 0.0064039206)),
+  nav_cam_to_sci_cam_transform = transform(vec3(-0.076526224, 0.011869553, 0.045409155), quat4(-0.0035414316, 0.0048089362, -0.0071515076, 0.99995659)),
+  nav_cam_to_haz_cam_timestamp_offset = -0.02,
```

Comment:
I don't think the geometry section is the best place for time offsets. Maybe in the calibrations section would be better? Or if you have another idea.